Wiki.md

```diff
@@ -0,0 +1,117 @@
+## Intro
+[Error Prone](https://github.com/google/error-prone) check to detect missing AutoDispose scope within defined scoped elements.
+
+## Installation
+
+Here are sample configurations which pulls in both the ErrorProne and the AutoDispose check.
+### Gradle
+```gradle
+  buildscript {
+    repositories {
+      maven {
+        url "https://plugins.gradle.org/m2/"
+      }
+    }
+  }
+  
+  plugins {
+    // we assume you are already using the Java plugin
+    id "net.ltgt.apt" version "0.13"
+    id "net.ltgt.errorprone" version "0.0.13"
+  }
+  
+  dependencies {
+    apt "com.uber.autodispose:autodispose-error-prone-checker:x.y.z" // where x.y.z is the latest version.
+  
+    errorprone "com.google.errorprone:error_prone_core:2.1.3"
+  }
+  
+  tasks.withType(JavaCompile) {
+    // Only if you want to support custom configuration
+    // Below is a sample configuration which include Conductor and Activity
+    options.compilerArgs += ["-XepOpt:AutoDisposeLeakCheck"
+                                         + "=com.bluelinelabs.conductor.Controller,android.app.Activity"]
+  }
+```
+
+### Maven
+```xml
+  <build>
```

Comment:
Move all this over one tab to the left. We don't need to have everything indented by two spaces automatically. Same with the gradle snippet above

Reply:
https://github.com/uber/AutoDispose/pull/162/commits/edba53cc061fb8f4d7e2dbe2936c7f329f0c2d96